app/grants/management/commands/grants_cv.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+"""
+
+Copyright (C) 2021 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from pprint import pprint
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+import requests
+from grants.models import Grant
+
+
+class Command(BaseCommand):
+
+    help = 'pulls data from https://twitter.com/austingriffith/status/1529825114597908484 and inserts into db'
+
+    def handle(self, *args, **options):
+
+        def run_query(q):
+            url = 'https://api.thegraph.com/subgraphs/name/danielesalatti/gtc-conviction-voting-rinkeby'
+            if not settings.DEBUG:
+                url = 'https://api.studio.thegraph.com/query/20308/gtc-conviction-voting-mainnet/v0.0.1'
+            request = requests.post(url,
+                                    '',
+                                    json={'query': q})
+            if request.status_code == 200:
+                return request.json()
+            else:
+                raise Exception('Query failed. return code is {}.      {}'.format(request.status_code, query))
+
+        queryGrants = """
+        query getGrants {
+          grants(orderBy: id, orderDirection: asc, first: 100) {
+            id
+            votes {
+              id
+              amount
+              createdAt
+            }
+            releases {
+              id
+              amount
+              createdAt
+            }
+          }
+        }
+        """
+
+        grantsResult = run_query(queryGrants)
+
+        ## TODO: calculation of voting power per grant
```

Comment:
what is expected here ? is this something which is yet to be added ?

Reply:
pinging him now

Reply:
I ported the logic from JS to Python here: https://github.com/DanieleSalatti/gtc-data-fetcher
Caveat: I've used Python a handful of times and it was mostly to fix existing scripts, so... 🙂

I am porting it over here too, how can I test this though?

Reply:
dani : youd have to setup the gitcoinco/web repo and run `./manage.py grants_cv`

everyone:
i've already tested it locally on mine and it works fine (at least with the graphs coded into this PR).

i dont know how to deploy it with the new path to production tho.  nor do i have knowledge of the existing code deployment process.

Reply:
Once it passes code review. I'll take care of it